### PR TITLE
Update to use labelmerge v0.4.3

### LINKED ISSUE #48: Update labelmerge usage
## The problem

With `labelmerge>=0.3` , there is a new feature to remove labels from being used in the merge. Ideally would like to use this to replace the `rm_bb_thal` rule, which would also allow us to slim down the container as FSL would no longer be necessary.

## Environment

* Operating System
* DBSC version

### PR DESCRIPTION
**Pull request by kaitj**
## Proposed changes

This PR resolves #48. 

Updates the workflow to take advantage of the drop label feature in the new `labelmerge` version. Consequently, removes the fsl stage and associated environment variables from the Dockerfile, which will slim the container down.

Took advantage of the PR to include a find-and-replace to automatically update the `scattr` container version that is used within the workflow (similar updating to `pyproject.toml` and `pipeline_description.json`.

## Types of changes

What types of changes does your code introduce? Put an `x` in the boxes that apply

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other (if none of the other choices apply)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you are unsure about any of the choices, don't hesitate to ask!

- [x] Changes have been tested to ensure that fix is effective or that a feature works.
- [ ] Changes pass the unit tests
- [ ] Code has been run through the `poe quality` task
- [ ] I have included necessary documentation or comments (as necessary)
- [x] Any dependent changes have been merged and published

## Notes
All PRs will undergo the unit testing before being reviewed. You may be requested to explain or make additional changes before the PR is accepted.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -69,7 +62,7 @@ RUN cd /opt/scattr \
 
 # Stage: runtime
 # NOTE: use fsl stage for library dependencies
```
Comment (tkkuehn) [inline Dockerfile:64]: Get rid of this comment

File: scattr/resources/zona_bb_subcortex/desc-ZonaBB_dseg.tsv [hunk 2]
```diff
@@ -13,11 +13,13 @@ index	name
...
 14	Right GPi
-15	Left Hippocampus
-16	Right Hippocampus
-17	Left NAc
-18	Right NAc
-19	Left Amygdala
-20	Right Amygdala
-21	Left ZI
-22	Right ZI
+15  Left Thalamus
```
Comment (tkkuehn) [inline scattr/resources/zona_bb_subcortex/desc-ZonaBB_dseg.tsv:16]: I think 15 and 16 here should be separated by tabs and aren't

**Review by tkkuehn — CHANGES_REQUESTED** [review]
A couple of small changes, then this'll be good to go (with the caveat that I didn't try it myself)

File: scattr/resources/zona_bb_subcortex/desc-ZonaBB_dseg.tsv (hunk 2 above)
Comment (kaitj) [reply scattr/resources/zona_bb_subcortex/desc-ZonaBB_dseg.tsv:16]: Ahh, consequence of vscode. Will fix this.

**Review by tkkuehn — APPROVED** [review]

**Pull request merged**